### PR TITLE
Add endpoint to retrieve bids by cp_id

### PR DESCRIPTION
**Pull request by mjoyce91**
Example: `localhost:3000/bids/cycleposition?cp_id=60000`

Will make another PR for all the necessary employee fields, as well as sorting.

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -64,6 +64,14 @@ var appRouter = function (app) {
...
   });
 
+  app.get("/bids/cycleposition", async function (req, res) {
+    let isCDO = false;
+    if (!req.headers.jwtauthorization) {
+      res.status(200).send({ Data: null, usl_id: 4000004, return_code: -1 })
+    }
+    res.status(200).send(await bidding.get_bids_by_cp(req.query));
+  });
+
```
Comment (scott062) [inline src/routes.js:74]: cyclePositions/bidders - from Pat's email (still theoretical until we get it though)

File: src/services/bids.js [hunk 2]
```diff
@@ -54,30 +75,39 @@ const get_delete_ind = id => (
 // Whether or not a CDO bid on the position
 const get_cdo_bid = id => ( { cdo_bid: 'N' } )
 
-const formatData = (data, isCDO = true) => {
+const formatData = (data, isCDO = true) => {s
```
Comment (scott062) [inline src/services/bids.js:78]:
```suggestion
const formatData = (data, isCDO = true) => {
```

Comment (scott062) [reply src/services/bids.js:78]: Typo

File: src/services/bids.js [hunk 3]
```diff
@@ -26,6 +26,27 @@ async function get_bid(cp_id, perdet_seq_num) {
   }
 }
 
+async function get_bids_by_cp(query) {
+  const cp_id = _.get(query, 'cp_id');
```
Comment (scott062) [inline src/services/bids.js:30]: request_params.cp_id ?

Comment (mjoyce91) [reply src/services/bids.js:30]: Some endpoints don't use the `request_params` prefix

File: src/routes.js (hunk 1 above)
Comment (mjoyce91) [reply src/routes.js:74]: Will address in #111

**Comment by mjoyce91** [comment]
@scott062 - I'll address changes in #111

**Pull request closed without merging**